cpp/oneapi/dal/algo/knn/backend/cpu/train_kernel_kd_tree.cpp

```diff
@@ -63,18 +63,19 @@ static train_result call_daal_kernel(const context_cpu& ctx,
         dummy_seed,
         desc.get_data_use_in_model() ? daal_knn::doUse : daal_knn::doNotUse);
 
-    const auto model_ptr = daal_knn::Model::create(column_count, NULL);
-    //if(!model_ptr)
-    //    bad alloc
+    const daal::algorithms::classifier::ModelPtr model_ptr = daal_knn::Model::create(column_count, NULL);
```

Comment:
If I'm not mistaken `NULL` usage can cause some warnings. Isn't it?

Reply:
Use `nullptr`, never use `NULL` in the new part.

Reply:
Will be changed when error handling is in place